partials/time_to_retirement.html

```diff
@@ -34,102 +34,105 @@
     </div>
     <div class="container-fluid output-controls-section">
         <div class="row">
-            <div class="graph-section supporting-content">
+            <div class="graph-section supporting-content col-xs-12">
                 <div class="supporting-content-wrapper">
                     <div class="col-md-8" id="graph-wrapper">
                         <div id="retirement-graph"></div>
                         <div id="explain-text">Financial independence is based on <a href="#/about">the 4% Safe Withdrawal Rate</a></div>
                     </div>
-                    <div class="supporting-section-body col-md-4">
-                        <div id="expenses-control" class="input-control col-md-offset-3 col-md-5">
-                            <div class="">
-                            	<div class="">
-						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="expenses = expenses + 100"></button>
+
+                    <div class="output-control-container col-xs-12 col-md-4">
+                        <div id="expenses-control" class="input-control primary-control col-xs-offset-3 col-xs-6 col-md-offset-3 col-md-6">
+                            <div>
+                            	<div>
+						        	<button class="btn btn-default glyphicon-plus control-increase-button control-change-button" type="button" ng-click="incrementOutputValue('expenses', 100)"></button>

```

Comment:
You might be able to write this as `ng-click="incrementOutputValue(expenses, 100)"` (note the lack of parentheses around `expenses`, meaning it's actually passing the scope variable `expenses` instead of a string `"expenses"`. But it might not let you. If it does, then that big if statement in the controller can be removed.